slave.mk

```diff
@@ -51,7 +51,7 @@ distclean : .platform clean
 include $(RULES_PATH)/config
 include $(RULES_PATH)/functions
 include $(RULES_PATH)/*.mk
-ifneq ($(CONFIGURED_PLATFORM), undefined)
+ifneq ($(wildcard $(PLATFORM_PATH)),)
```

Comment:
if the .platform file exists but empty, the make command will fail as the $(CONFIGURED_PLATFORM) is not undefined. With this fix, the make command will not fail unexpectedly.